dataland-rabbitmq/definitions.json

```diff
@@ -0,0 +1,91 @@
+{
+  "rabbit_version": "3.7.28",
+  "users": [
+    {
+      "name": "RABBITMQ_USER_PLACEHOLDER",
+      "password_hash": "RABBITMQ_PASS_HASH_PLACEHOLDER",
+      "hashing_algorithm": "rabbit_password_hashing_sha256",
+      "tags": "administrator"
+    }
+  ],
+  "vhosts": [
+    {
+      "name": "vhost"
+    }
+  ],
+  "permissions": [
+    {
+      "user": "RABBITMQ_USER_PLACEHOLDER",
+      "vhost": "vhost",
+      "configure": ".*",
+      "write": ".*",
+      "read": ".*"
+    }
+  ],
+  "topic_permissions": [],
+  "parameters": [],
+  "global_parameters": [
+    {
+      "name": "cluster_name",
+      "value": "rabbit@17805b5a5151"
+    }
+  ],
+  "policies": [],
+  "queues": [
+    {
+    "name": "deadLetterQueue",
+    "vhost": "vhost",
+    "durable": true,
+    "auto_delete": false,
+    "arguments": {}
+  }
+  ],
+  "exchanges": [
+    {
+      "name": "dataReceived",
+      "vhost": "vhost",
+      "type": "direct",
+      "durable": true,
+      "auto_delete": false,
+      "internal": false,
+      "arguments": {}
+    },
+    {
+      "name": "dataQualityAssured",
+      "vhost": "vhost",
+      "type": "direct",
+      "durable": true,
+      "auto_delete": false,
+      "internal": false,
+      "arguments": {}
+    },
+    {
+      "name": "dataStored",
+      "vhost": "vhost",
+      "type": "direct",
+      "durable": true,
+      "auto_delete": false,
+      "internal": false,
+      "arguments": {}
+    },
+    {
+      "name": "deadLetter",
+      "vhost": "vhost",
+      "type": "direct",
+      "durable": true,
+      "auto_delete": false,
+      "internal": false,
+      "arguments": {}
+    }
+  ],
+  "bindings": [
+    {
+      "source": "deadLetter",
```

Comment:
Haben wir mal beobachtet dass die Messages auch hierhin laufen?

Reply:
Ja, haben wir manuell gemacht. Auch die messages die auf der dead letter queue liegen in der rabbitmq gui angeschaut.

Reply:
perfekt.